<!-- ROS launch file -->
<launch>
    <arg name="node_name"       default="watchdog" />

    <arg name="status_o"        default="$(arg node_name)/status" />
    <arg name="status_service_o"        default="$(arg node_name)/status" />

    <node pkg="ros_watchdog" type="RosWatchdog.py" name="watchdog" output="screen">
        <param name="topics_cfg_file" value="$(find ros_watchdog)/scripts/topics.ini"/>
        <param name="sensors_cfg_file" value="$(find ros_watchdog)/scripts/sensors.ini"/>
        <param name="topic_check_rate" value="1.0"/>
        <param name="bVerbose" value="True"/>

        <remap from="/status" to="$(arg status_o)"/>
        <remap from="/status_service" to="$(arg status_service_o)"/>
    </node>
</launch>
